website/static/js/comment.js

```diff
@@ -77,19 +261,50 @@ var BaseComment = function() {
 
     self.replying = ko.observable(false);
     self.replyContent = ko.observable('');
+    self.replyMentions = ko.observableArray([]);
+
+    self.saveContent = ko.computed(function() {
+        var content = self.replyContent();
+        if (!content) {
+            content = '';
+        }
+        self.replyMentions([]);
+        var regex = /<span.*?data-atwho-guid="([a-z\d]{5})".*?>((@|\+)[\w\s]+)<\/span>/;
+        var matches = content.match(/<span.*?data-atwho-guid="([a-z\d]{5})".*?>((@|\+)[\w\s]+)<\/span>/g);
+        if (matches) {
+            for (var i = 0; i < matches.length; i++) {
+                var match = regex.exec(matches[i]);
+                var guid = match[1];
+                var mention = match[2];
+                var url = '/' + guid + '/';
+                content = content.replace(match[0], '['+ mention + '](' + url + ')');
+
+                if (guid && self.replyMentions.indexOf(guid) === -1) {
+                    self.replyMentions.push(guid);
+                }
+            }
+        }
+        return content.replace(/<br>/g, '&#13;&#10;');
```

Comment:
Comments are rendered as markdown which requires two returns to display as a line break. From what I understand this is normal markdown behavior.